modules/gapi/src/compiler/gislandmodel.hpp

```diff
@@ -290,7 +292,8 @@ namespace GIslandModel
     //     from the original model (! don't mix with DataSlot)
     // FIXME: GAPI_EXPORTS because of tests only!
     ade::NodeHandle GAPI_EXPORTS producerOf(const ConstGraph &g, ade::NodeHandle &data_nh);
-
+    // traceName - returns pretty island name for passed island node
```

Comment:
I believe in should be  `traceIslandName`

Reply:
Thanks, Anton!!